### PR TITLE
changed dark theme + moved css from index.html to own folder

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -969,7 +662,12 @@
         <div class="header item">FFX|V Fish Tracker App</div>
         <div class="right item">
           <b>Current Time:</b>&nbsp;<span id="eorzeaClock">##:##</span>&nbsp;(<span id="earthClock">#########</span>)
-          <button id="theme-button" class="theme-button"></button>
+          <div class="theme-switch-wrapper">
```
Comment (icykoneko) [inline index.html]: I would prefer this use the Semantic UI instead.
See https://semantic-ui.com/modules/checkbox.html#toggle

Comment (Akurosia) [reply index.html]: done

**Comment by Akurosia** [comment]
fixed the hover over color (as there was on overwrite in the semantic for the elements which needs to be overwritten for dark theme)

**Comment by icykoneko** [comment]
Sorry, I was away this week, I'll check over this after tonight's events.

**Pull request merged**